server/routes/delete-event.js

```diff
@@ -1,4 +1,6 @@
 var deleteEvent = require('../db/delete-event.js');
```

Comment:
The naming convention isn't quite right here: routes are `my-file.js` whilst functions are `myFile.js`